.github/workflows/tests/test_raster.py

```diff
@@ -3,6 +3,9 @@
 import httpx
 
 raster_endpoint = "http://0.0.0.0:8082"
+seeded_collection = "noaa-emergency-response"
+seeded_id = "20200307aC0853900w361030"
+z, x, y = 15, 8589, 12849
```

Comment:
Thanks for implementing this - would this PR be a good time to define these constants as [fixtures](https://docs.pytest.org/en/stable/how-to/fixtures.html)? Later on, when we implement more automated integration tests, we could use these to test live environments by overriding the endpoint and items for the tests.

Reply:
Thanks for the guidance. I think the updates are there now